dexs/bluefin/index.ts

```diff
@@ -0,0 +1,62 @@
+import { BreakdownAdapter } from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import axios from "axios";
+import BigNumber from "bignumber.js";
+
+const url_config = {
+  "arb": "https://dapi.api.arbitrum-prod.firefly.exchange/marketData",
+  "sui": "https://dapi.api.sui-prod.bluefin.io/marketData",
+}
+
+const product_config = {
+  "arb": ["ETH-PERP", "BTC-PERP"],
+  "sui": ["ETH-PERP", "BTC-PERP"],
+}
+
+const fetchURL = (baseURL: string, product: string) => {
+  baseURL = `${baseURL}` + `?symbol=${product}`;
+  return baseURL;
+}
+
+const computeVolume = async (timestamp: number, baseUrl: string, productIds: string[]) => {
```

Comment:
Please define a return type for the method.  It is best to create an interface at the top for the return type:
```
return {
    totalVolume: undefined,
    dailyVolume: dailyVolume ? `${dailyVolume}` : undefined,
    timestamp: timestamp,
  };
  ```
  Something like:
  ```
  interface Volume {
     totalVolume: number | undefined,
     dailyVolume: number | undefined,
     timestamp: number,     
  }
  ```